src/apprt/gtk/version.zig

```diff
@@ -26,6 +26,16 @@ pub inline fn atLeast(
     // If we're in comptime then we can't check the runtime version.
     if (@inComptime()) return true;
 
+    return runtimeAtLeast(major, minor, micro);
+}
+
+/// Verifies that the GTK version at runtime is at least the given
+/// version.
```

Comment:
I think documentation here and in `atLeast` could be better to clarify when either should be used. I'm happy to cover that myself but for example clarifying that:

* `atLeast` should be used when there is a symbol being linked
* `runtimeAtLeast` should be used when there is a _behavior_ being checked.

Reply:
Added clarification on when each function should be used. Please let me know if they are inappropriate.